float32 x
float32 y
float32 z
float32 finger_percentage
---
string plan_result

